src/pages/iou/steps/MoneyRequestAmountForm.js

```diff
@@ -171,6 +176,22 @@ function MoneyRequestAmountForm({amount, currency, isEditing, forwardedRef, onCu
         onSubmitButtonPress(currentAmount);
     }, [onSubmitButtonPress, currentAmount]);
 
+    /**
+     * Input handler to check for a forward-delete key (or keyboard shortcut) press.
+     */
+    const textInputKeyPress = ({nativeEvent}) => {
+        const key = nativeEvent.key.toLowerCase();
+        if (Browser.isMobileSafari() && key === 'control') {
```

Comment:
`'control'` -> `CONST.PLATFORM_SPECIFIC_KEYS.CTRL.DEFAULT` ?

Reply:
Make sense. I've seen these key constants in the past, but then I forgot all about them. It's fixed now.